<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrationWindow</class>
 <widget class="QDialog" name="RegistrationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Register</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="backgroundLabel">
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/images/netflix.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QWidget" name="overlayWidget">
     <layout class="QVBoxLayout" name="mainLayout">
      <item>
       <spacer name="verticalSpacerTop">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint">
         <size>
          <width>20</width>
          <height>60</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="centerLayout">
        <item>
         <spacer name="horizontalSpacerLeft">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QFrame" name="formPanel">
          <property name="styleSheet">
           <string notr="true">
                background-color: rgba(0, 0, 0, 0.80);
                border-radius: 16px;
                padding: 40px 32px 32px 32px;
           </string>
          </property>
          <layout class="QVBoxLayout" name="formLayout">
           <item>
            <widget class="QLabel" name="titleLabel">
             <property name="text">
              <string>Sign Up</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="styleSheet">
              <string notr="true">
                color: white;
                font-size: 28px;
                font-weight: bold;
                margin-bottom: 20px;
              </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="usernameLineEdit">
             <property name="placeholderText">
              <string>Username</string>
             </property>
             <property name="styleSheet">
              <string notr="true">
                background-color: #222;
                border: none;
                border-radius: 6px;
                padding: 14px;
                color: white;
                font-size: 15px;
                margin-bottom: 12px;
              </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="passwordLineEdit">
             <property name="placeholderText">
              <string>Password</string>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
             <property name="styleSheet">
              <string notr="true">
                background-color: #222;
                border: none;
                border-radius: 6px;
                padding: 14px;
                color: white;
                font-size: 15px;
                margin-bottom: 12px;
              </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="confirmPasswordLineEdit">
             <property name="placeholderText">
              <string>Confirm Password</string>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
             <property name="styleSheet">
              <string notr="true">
                background-color: #222;
                border: none;
                border-radius: 6px;
                padding: 14px;
                color: white;
                font-size: 15px;
                margin-bottom: 12px;
              </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="registerButton">
             <property name="text">
              <string>Sign up</string>
             </property>
             <property name="styleSheet">
              <string notr="true">
                background: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #3d5af1, stop:1 #00c6fb);
                color: white;
                font-weight: bold;
                font-size: 17px;
                border: none;
                padding: 14px;
                border-radius: 6px;
                margin-top: 8px;
                margin-bottom: 10px;
              </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="signinLabel">
             <property name="text">
              <string>Already have an account?</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="styleSheet">
              <string notr="true">
                color: #cccccc;
                font-size: 14px;
                margin-top: 10px;
              </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="signinButton">
             <property name="text">
              <string>Sign in</string>
             </property>
             <property name="styleSheet">
              <string notr="true">
                background: transparent;
                color: white;
                font-size: 14px;
                border: none;
                text-decoration: underline;
                font-weight: bold;
              </string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacerRight">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacerBottom">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint">
         <size>
          <width>20</width>
          <height>60</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
